src/tracks/Pattern.cpp

```diff
@@ -938,9 +955,74 @@ void PatternView::paintEvent( QPaintEvent * )
 
 	const int x_base = TCO_BORDER_WIDTH;
 
-	// melody pattern paint event
 	NoteVector const & noteCollection = m_pat->m_notes;
-	if( m_pat->m_patternType == Pattern::MelodyPattern && !noteCollection.empty() )
+
+	// Beat pattern paint event
```

Comment:
Sure, I ended up not doing it because I moved a big block but didn't change it and wasn't sure how it would show in the diff. Can I leave this style fix for after it's decided whether this will be merged before or after refactor? Because if it's merged after I'm probably going to have to rebase against the fixed-style master anyways

Reply:
I'm okay with leaving for clang PR.